.devcontainer/devcontainer.json

```diff
@@ -1,16 +1,13 @@
 {
   "name": "traffic development environment",
   "image": "ghcr.io/xoolive/traffic/poetry:latest",
-
   "customizations": {
-    // Configure properties specific to VS Code.
     "vscode": {
-      // Add the IDs of extensions you want installed when the container is created.
       "extensions": [
         "ms-python.python",
         "streetsidesoftware.code-spell-checker"
       ]
     }
   },
-  "postCreateCommand": "poetry install"
+  "postCreateCommand": "poetry install && poetry shell"
```

Comment:
If I understand well, the `poetry shell` replaces the `virtualenvs.create = False` ?

Reply:
@xoolive, do you have any idea when you'll have time to look into this?